jupyterlab-mlops/mlflow/start-mlflow.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/bash -i
+
+MLFLOW_HOME=/home/jovyan/mlflow
+
+mkdir -p ${MLFLOW_HOME}
+mlflow db upgrade sqlite:///${MLFLOW_HOME}/mlflow.db >> /tmp/mlflow.logs 2>&1
```

Comment:
How does it relate to https://github.com/mlflow/mlflow/issues/4762#event-5282630805 ?

Reply:
Thanks, it looks like `db upgrade` is no longer needed in 1.20.2. Removed.